lxd/daemon.go

```diff
@@ -1453,7 +1454,7 @@ func (d *Daemon) init() error {
 	}
 
 	/* Setup the web server */
-	config := &endpoints.Config{
+	endpointsConfig := &endpoints.Config{
```

Comment:
can the variable clarifications be moved to separate commits/PRs to isolate the feature changes from the clean up changes?